### PR TITLE
#167 removed not renderable buttons

### PR DESCRIPTION
**Pull request by AlexeyALeonov**

### CODE REVIEW

**Review by heunland — CHANGES_REQUESTED** [review]
there is a word misspelled in line 118, please fix:
`**Username** filed` should say `**Username** field`

File: resources/faq/how-to-remote-access-the-web-dashboard.md [hunk 1]
```diff
@@ -115,7 +115,7 @@ Tap on plus sign and select **New Identity**.
 
 #### New Identity
 
-Specify a name of the Identity in the field **Name**, provide a username for your PC with storagenode in the **Username** filed, specify a password in the **Password** field, confirm creation with a confirmation $$\sqrt{}$$ button.
+Specify a name of the Identity in the field **Name**, provide a username for your PC with storagenode in the **Username** filed, specify a password in the **Password** field, confirm creation.
```
Comment (AlexeyALeonov) [inline resources/faq/how-to-remote-access-the-web-dashboard.md]:
```suggestion
Specify a name of the Identity in the field **Name**, provide a username for your PC with storagenode in the **Username** field, specify a password in the **Password** field, confirm creation.
```

Comment (AlexeyALeonov) [reply resources/faq/how-to-remote-access-the-web-dashboard.md]: I just fixed it in the next commit, so I you can just close this thread.

Comment (heunland) [reply resources/faq/how-to-remote-access-the-web-dashboard.md]: I don't see any resolve conversation button below

Comment (AlexeyALeonov) [reply resources/faq/how-to-remote-access-the-web-dashboard.md]: Seems it's related to rights. Perhaps @profclems can help you.

Comment (AlexeyALeonov) [reply resources/faq/how-to-remote-access-the-web-dashboard.md]: I resolved this thread if you don't mind.

**Pull request merged**